### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by AFulgens**
Fixes include:

- Fixing some typos
- Using NAmE instead of BrE consistently
- Making capitalization of items, units, etc. consistent

Also added a first version of a personal dictionary for spellchecking.

Spellchecking done with
```bash
find . -type f -name "*.tex" -exec aspell -d en_US -p=./.aspell.homm3.pws --mode=tex --dont-backup check {} \;
```

### CODE REVIEW

**Review by qwrtln — CHANGES_REQUESTED** [review]
Thank you for this, it's looking superb. Please update the workflows running on PR checks to include the spell check. My recommendation would be to add it to `build-components-list.yaml` and `build-rule-book.yaml`. An additional step before `Complie LaTeX document`.

**Comment by AFulgens** [comment]
Adding the spellcheck to the workflow is a bit more involved, because `aspell`, even when running with `--mode=tex` will flag a lot of the LaTeX commands, because they are by default not ignored. It will thus catch all things like "px" (for sizing), "svg" (for filenames), "sep" (in "sep=0"), etc.

In case you want to, I can add that in a later PR or block this until I figured it all out. I'll also have to check whether aspell must be installed into the GitHub runner image or it's already there.

**Comment by qwrtln** [comment]
There is a GitHub action with aspell: https://github.com/reitermarkus/aspell
I'm not sure how useful it is, because it's the first time I'm hearing about this tool. If it works like that in the workflow, how is it better when running locally?

**Comment by AFulgens** [comment]
It's effectively an interactive spellchecker on the command line, see  this screenshot:
image

I've looked now at that action and the problem is exactly what's also seen in the screenshot: not all TeX commands are interpreted "correctly" by the tool with the default settings. One would have to go through and add the correct configuration for the commands used in this project (see the documentation of the tool: http://aspell.net/man-html/The-Options.html#TeX_002fLaTeX-Filter ).

What I did locally is I've run the interactive checker through all files and I've hit "Ignore" every time it was flagging a LaTeX command parameter. Not really automation friendly at this point in time 🙃

**Review by qwrtln — APPROVED** [review]
I see. Well, OK. Thank you for this contribution. Please add a note about aspell in the readme file in `Local Development` section. I'll see if I can come up with some magic to run this in the CI.

**Comment by qwrtln** [comment]
Ideally, put the command from the first comment 😁

**Comment by AFulgens** [comment]
Added a new commit with the enhancement to the README. You don't have to look into how to automate that with GitHub, I'll do some research and will come with an additional PR.

File: README.md [hunk 1]
```diff
@@ -74,6 +75,24 @@ Then, use the script:
 ./make_printable.sh
 ```
 
+### 🔎 Spellchecking
+
+TeXstudio has built-in spellchecking, but the first steps have been made towards automated spellchecking with aspell. For local development, after installing the tool, you can run it from the command line for example with
```
Comment (qwrtln) [inline README.md:80]:
```suggestion
TeXstudio has built-in spellchecking, but the first steps have been made towards automated spellchecking with aspell.
For local development, after installing the tool, you can run it from the command line for example with
```

File: README.md [hunk 2]
```diff
@@ -74,6 +75,24 @@ Then, use the script:
...
+aspell -d en_US -p=./.aspell.homm3.pws --mode=tex --dont-backup check main.tex
+```
+
+or when wanting to check all `.tex` files then with
+
+```bash
+find . -type f -name "*.tex" -exec aspell -d en_US -p=./.aspell.homm3.pws --mode=tex --dont-backup check {} \;
+```
+
+Please note that currently the tool will flag many parameters in LaTeX commands. We are currently looking into it, how best to remediate this.
```
Comment (qwrtln) [inline README.md:92]:
```suggestion
Please note that currently the tool will flag many parameters in LaTeX commands.
We are currently looking into it, how best to remediate this.
```
Please keep it one sentence per line.

File: README.md [hunk 3]
```diff
@@ -74,6 +75,24 @@ Then, use the script:
 ./make_printable.sh
 ```
 
+### 🔎 Spellchecking
```
Comment (qwrtln) [inline README.md:78]: Nice choice of emoji 😆

File: README.md (hunk 2 above)
Comment (AFulgens) [reply README.md:92]: Is `git amend` + `force-push` OK for you for the changes in the README or would you prefer a separate commit?

Comment (AFulgens) [reply README.md]: Done.

**Review by qwrtln — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**